catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_s2ls_intertidal_cyear_3.jsonnet

```diff
@@ -0,0 +1,135 @@
+          [Band details](https://knowledge.dea.ga.gov.au/data/product/dea-intertidal/?tab=details#core-product-layers)
+        |||,
+        'gee:units': units.meter,
+      },
+      {
+        name: 'elevation_uncertainty',
+        description: |||
+          DEA Intertidal Elevation Uncertainty provides a measure of the quality of each modelled elevation value in metre units. Uncertainty is calculated by assessing how cleanly the modelled elevation separates satellite observations into dry and wet observations. This is achieved by identifying satellite observations that were misclassified by the modelled elevation (for instance, pixels that were observed as wet at tide heights lower than the modelled elevation, or alternately, observed as dry at higher tide heights). The spread of tide heights from these misclassified observations is summarised using a robust Median Absolute Deviation (MAD) statistic, and reported as 0.5 x MAD to represent one-sided uncertainty bounds 
+          (i.e. ± uncertainty on either side of the pixel's elevation). Common causes of high elevation uncertainty can be poor tidal model performance, rapidly changing intertida morphology, or noisy underlying satellite data.
+          [Band details](https://knowledge.dea.ga.gov.au/data/product/dea-intertidal/?tab=details#core-product-layers)
+        |||,
+        'gee:units': units.meter,
+      },
+      {
+        name: 'exposure',
+        description: |||
+          DEA Intertidal Exposure models the percentage of time that any intertidal pixel of known elevation is exposed from tidal inundation. Exposure is calculated by comparing the pixel elevation back against a high temporal resolution model of tide heights for that location, based on the Ensemble Tidal Modelling approach. Exposure percentage is calculated as the fraction of exposed observations relative to the total number of observations generated in the high temporal resolution tidal model for the 3-year product epoch.
+          [Band details](https://knowledge.dea.ga.gov.au/data/product/dea-intertidal/?tab=details#core-product-layers)
+        |||,
+        'gee:units': units.meter,
+      },
+    ],
+    elevation: {minimum: -5, maximum: 5, 'gee:estimated_range': true},
+    elevation_uncertainty: {minimum: 0, maximum: 1.0, 'gee:estimated_range': true},
+    exposure: {
+      minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    'gee:visualizations': [
+      {
+        display_name: 'DEA Intertidal',
+        lookat: {lon: 133.88, lat: -23.70, zoom: 5},
```

Comment:
These coordinates are pointing to the center of Australia. Please correct them to match the preview script.